src/logic/collectibles/utils/index.ts

```diff
@@ -64,6 +63,11 @@ export const getERC721Symbol = async (contractAddress: string): Promise<string>
   return tokenSymbol
 }
 
+export const isENSContract = (contractAddress: string): boolean => {
+  const ENS_TOKEN_CONTRACT = '0x57f1887a8BF19b14fC0dF6Fd9B2acc9Af147eA85'
```

Comment:
For which network is this ENS contract address?

Reply:
For both rinkeby and mainnet, not sure if for the deployed ENS contract for other networks we have this issue with the symbol() method too. In any case would be better to have support for ewc/xDai in this method, do you have those addresses for adding them?

Reply:
For xDai I don't think we have it yet.
For EWC and Volta I think it should be the ETHBaseRegister ones:
https://energyweb.atlassian.net/wiki/spaces/EWF/pages/555745281/Using+ENS

Reply:
I'm thinking on adding the addresses on the config file what do you think?

Reply:
Personally, I prefer the approach I took in https://github.com/gnosis/safe-react/blob/1587-ens-mislabeled-as-eth/src/logic/collectibles/utils/index.ts#L16-L19

It's so specific that populating the config files with that info will make them unnecessary complex in the future.

Reply:
That makes sense, will do that 👍

Reply:
I added ens addresses for both volta and ewc too 👍